ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractIdIT.scala

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.grpc.{GrpcException, GrpcStatus}
+import com.daml.ledger.api.refinements.ApiTypes.Party
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions.{assertGrpcError, fail}
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
+import com.daml.ledger.api.v1.value.{Record, RecordField, Value}
+import com.daml.ledger.client.binding.Primitive.ContractId
+import com.daml.ledger.test.semantic.ContractIdTests._
+import io.grpc.Status
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+// Check the Ledger API accepts or rejects non-suffixed contract ID.
+// - Central committer ledger implementations (sandboxes, KV...) may accept non-suffixed CID
+// - Distributed ledger implementations (e.g. Canton) must reject non-suffixed CID
+final class ContractIdIT extends LedgerTestSuite {
```

Comment:
maybe include `Suffixing` in the name to make it clear what we’re testing here.

Reply:
I was think we could add inside tests for V0 contract ID.